yarn.lock

```diff
@@ -2950,6 +2965,11 @@ csstype@^2.2.0:
   resolved "https://registry.yarnpkg.com/csstype/-/csstype-2.6.10.tgz#e63af50e66d7c266edb6b32909cfd0aabe03928b"
   integrity sha512-D34BqZU4cIlMCY93rZHbrq9pjTAQJ3U8S8rfBqjwHxkGPThWFjzZDQpgMJY0QViLxth6ZKYiwFBo14RdN44U/w==
 
+csstype@^3.0.2:
```

Comment:
Not sure why this was added 🤔 
We already had a PR to update the lock from dependabot but I think we can include it in this PR

Reply:
Hmm I have no idea also ahah If you want me to revert it I can, if not, I can just leave it!

Reply:
Just keep it like this, yarn knows what he's doing.
Could you just run `yarn why csstype` and post the result here?

Reply:
Ahh it's a dep of `@types/react` and `@types/react-dom`

```
yarn why v1.22.5
[1/4] 🤔  Why do we have the module "csstype"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "csstype@3.0.4"
info Reasons this module exists
   - "@types#react" depends on it
   - Hoisted from "@types#react#csstype"
info Disk size without dependencies: "1.1MB"
info Disk size with unique dependencies: "1.1MB"
info Disk size with transitive dependencies: "1.1MB"
info Number of shared dependencies: 0
=> Found "@types/react-dom#csstype@2.6.10"
info Reasons this module exists
   - "@types#react-dom#@types#react" depends on it
   - Hoisted from "@types#react-dom#@types#react#csstype"
info Disk size without dependencies: "1.8MB"
info Disk size with unique dependencies: "1.8MB"
info Disk size with transitive dependencies: "1.8MB"
info Number of shared dependencies: 0
=> Found "@types/testing-library__react#csstype@2.6.10"
info Reasons this module exists
   - "@testing-library#react#@types#testing-library__react#@types#react-dom#@types#react" depends on it
   - Hoisted from "@testing-library#react#@types#testing-library__react#@types#react-dom#@types#react#csstype"
info Disk size without dependencies: "1.8MB"
info Disk size with unique dependencies: "1.8MB"
info Disk size with transitive dependencies: "1.8MB"
info Number of shared dependencies: 0
✨  Done in 0.56s.

```